docs/live-streaming.md

```diff
@@ -0,0 +1,157 @@
+should send it as fast as possible and then simply "sleep" for the remaining
+time! It means that you should split the whole 1/60s time **evenly** to all
+single 1316-byte units. So, there is a "unit duration" for every single
+1316-byte unit, so you send the unit, and then you should "sleep" **after every
+unit**, if sending (as it usually happens) has taken less time than the exact
+time predicted to be spent by sending this unit.
+
+Should that whole "I-Frame with corresponding audio" sending take **exactly**
+1/60s? Of course, not exactly. This may have some slight time differences,
+which come from two reasons:
+
+1. Sending over the network has only average time perdictability. Unusual and
+unexpected delays, coming from both the network and your system, may slightly
+destroy your perfect time calculations. Therefore you should not rely on the
+exact number of bytes sent by one "frame package", but rather on overall
+transmission size, and do often periodic synchronization of the transmission
+time basing on the timestamps in the stream.
+
+2. Not all frames as I-Frames. Most of the frames sent in the video stream are
+"difference frames" (P or B frames), which need all preceding (or even
+succeding) frames already received to be able to decode the designated frame.
+As you can guess, difference frames are much shorter than I-Frames, so it makes
+the whole "frame package" carry much less data to transport, but they still
+cover the same time gap ("duration"). 
+
+Taking all the above things into account, you know now that actually the most
+important in the synchronization is to make the I-Frame completely transported
+at specified time, and every next frame must be received at more-less the same
+time as I-Frame, next by one duration period, and with some only slight time
+tolerance, which results from size difference. what must be absolutely adhered
+to, however, is that the distance between the first network unit transporting
+the first portion of one I-Frame, and the same unit for the next I-Frame must
+correspond with the time distance between these two same I-frames that results
+from their timestamps.
+
+In other words: there is some slight buffering at the receiving side, but the
+requirement for a live streaming is that: data must be transmitted with exactly
+the same AVERAGE speed, as the video player that would play it, or even more
+exactly, the data must be produced exactly the same fast as they will be consumed
+by the video player.
+
+
+Live Streaming Process
+======================
+
+Now that you know how the Live Streaming should turn a bunch of MPEG-TS
+encoded video stream into a network live stream, let's complete the definition
+of the live streaming transport over the network using SRT.
+
+The source side should be a real live stream, that is, for example:
+
+- a file read by an application that can make a live stream out of it
+- a grab device that is grabbing the data in constant rates and then
```

Comment:
a capture device that is capturing data at constant rates and passing the data to to a live encoder

Reply:
I'd use rather "a grab device that is capturing data" (the "grab device" term is taken from the ffmpeg documentation).